src/generators/web.cr

```diff
@@ -105,10 +106,16 @@ class LuckyCli::Generators::Web
     DEPS_LIST
   end
 
-    private def ensure_directory_does_not_exist
-      if Dir.exists?("./#{project_dir}")
-        puts "Folder named #{project_name} already exists, please use a different name"
-        exit
-      end
+  private def add_env_to_spec_helper
+    prepend_text to: "spec/spec_helper.cr", text: <<-LUCKY_ENV
+    ENV["LUCKY_ENV"] = "test"
```

Comment:
I just realized that I deleted the newline characters I had in an earlier version.  Re-adding them now.